src/components/Button/Button.story.jsx

```diff
@@ -103,3 +104,25 @@ stories.add('Large Button', () => (
     },
   }
 )
+
+stories.add('Icon Button', () => (
+  <Button
+    size={select('Size', ['default', 'large'], 'default')}
+    color={select('Color', ['default', 'primary', 'secondary'], 'default')}
+    disabled={boolean('Disabled', false)}
+    type='button'
+  >
+    <span>
+      <Icon kind="edit" />
```

Comment:
Would be nice to have an additional knob here that we can specify the icon kind with.